OutSystems/outsystems-datagrid-reactive
# RGRIDT 849 - Defining column dependency - Other column types

### What was happening
*  Wasn't possible to bind an interaction of a cell with the content of another one in the same row.

### What was done
* A new method SetCellData() was created;

### Test Steps
1. Add an action on OnCellValueChange available on the column
2. Inside this action use the SetCellData client action to change a value of another cell inside this row.
3. Check if the second cell is edited, after the first one is edited. 


### Checklist
